ci/jenkins/jobs/macros.yaml

```diff
@@ -413,3 +413,11 @@
           done
 
           kubectl delete ns antrea-test || true
+- builder:
+    name: builder-elk-flow-collector
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          source /home/ubuntu/.bashrc
+          ./ci/test-elk-flow-collector.sh
```

Comment:
Could you add newline at end of file? https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Done. Thanks!